openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+{% spaceless %}
+<div id="openassessment__message" class="openassessment__message message">
+    <h3 class="message__title">
+        {% Assignment in progress %}
+    </h3>
+    <div class="message__content">
+        <p>
+            {% if training %}
+                {% if training_deadline_approaching %}
+                    {% trans "Learner training will close soon. " %}
```

Comment:
This was probably existing code, but relying on a translated message to end with a space (because of concatenation of other strings following) is a really bad idea. The translator has no way of knowing the space is important, and is likely to drop it. The best practice is to use placeholders, or just duplicate the second string (so there would be an if/else block with the second string appearing in both).